### DISCUSSION TITLE
Linked Image

### DISCUSSION THREAD

**Discussion by Ferdyverse** [Lednerb/bilberry-hugo-theme#456] [Q&A]
Hello,

I would like to use a linked image on my page. 

But the combination of markdown link an image is not working properly.
`[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/LyErffRX0HM/0.jpg)](http://www.youtube.com/watch?v=LyErffRX0HM)`

It renders like this:
![image](https://user-images.githubusercontent.com/1213924/205659735-ee538196-0937-43f9-ba7d-f985decf61dc.png)

Does someone know how I can fix this?

**Comment by igor-baiborodine** [comment]
@Brazier85, are you trying to embed a Youtube video? If yes, please read the [Video](https://github.com/Lednerb/bilberry-hugo-theme#video) section of README.
![image](https://user-images.githubusercontent.com/1027701/205706171-83838a5d-eb30-474a-b8b1-7e1d6d63c6ad.png)

**Reply by Ferdyverse** [reply]
No, this was just an example. Ich want to link a image like this in html

`<a href="mylink" target="_blank"><img src="/images/myimahe.png" /> </a>`

In my case I want to link a donation button image with ko-fi

**Reply by igor-baiborodine** [reply]
Is the page in question `content/page/about/index.md` : https://github.com/Brazier85/brazier85.de/commit/8d16efeaa1078f51a81ca4a566f769e14b0c3c8a?

**Reply by Ferdyverse** [reply]
@igor-baiborodine yes. The markdown link should look like this:
`[![Support me](/images/kofi_bg_tag_white.png)](https://ko-fi.com/brazier85)`
The version I added for now is a workaround

**Comment by Ferdyverse** [comment]
No ideas on how to fix that?

**Reply by igor-baiborodine** [reply]
It was fixed in release [v3.16.0](https://github.com/Lednerb/bilberry-hugo-theme/releases/tag/v3.16.0).
